packaging/bundle-ebpf.sh

```diff
@@ -11,4 +11,7 @@ curl -sSL --connect-timeout 10 --retry 3 "https://github.com/netdata/kernel-coll
 grep "${EBPF_TARBALL}" "${SRCDIR}/packaging/ebpf.checksums" | sha256sum -c - || exit 1
 tar -xaf "${EBPF_TARBALL}" -C "${SRCDIR}/tmp/ebpf" || exit 1
 # shellcheck disable=SC2046
-cp -a $(find "${SRCDIR}/tmp/ebpf" -mindepth 1 -maxdepth 1) "${PLUGINDIR}"
+if [ ! -d ${PLUGINDIR}/ebpf.d ];then
+    mkdir ${PLUGINDIR}/ebpf.d
+fi
+cp -a $(find "${SRCDIR}/tmp/ebpf" -mindepth 1 -maxdepth 1) "${PLUGINDIR}/ebpf.d"
```

Comment:
Unlike the other two cases, this is a false positive (we want word splitting to happen here).

The warning should be suppressed by adding a comment before this line reading `shellcheck disable=SC2046`.

Reply:
or switch to find + -exec command (mv)

Reply:
Hmm, yeah, that would probably be a better approach here. Avoiding the need to use command substitution is exactly what the `-exec` predicate is for.

Reply:
Given that this is a one-off code path though, I’m not hugely worried either way, just as long as the code works and shellcheck is not complaining about it.